tasks/adjust_config.yml

```diff
@@ -32,41 +32,21 @@
           cmd: |-
             docker exec --user={{ nextcloud_uid }}:{{ nextcloud_gid }} {{ nextcloud_identifier }}-server php /var/www/html/occ {{ item.appType }} --type={{ item.type }} --value={{ item.value }} -- {{ item.appConfig }} {{ item.configName }}
       with_items:
-        - appType: "config:app:set"
-          appConfig: "previewgenerator"
-          configName: "squareSizes"
-          type: string
-          value: "{{ nextcloud_preview_squareSizes }}"
-        - appType: "config:app:set"
-          appConfig: "previewgenerator"
-          configName: "widthSizes"
-          type: string
-          value: "{{ nextcloud_preview_widthSizes }}"
-        - appType: "config:app:set"
-          appConfig: "previewgenerator"
-          configName: "heightSizes"
-          type: string
-          value: "{{ nextcloud_preview_heightSizes }}"
         - appType: "config:system:set"
           appConfig: ""
           configName: "preview_max_x"
-          type: int
-          value: "{{ nextcloud_preview_preview_max_x }}"
+          type: "{{ 'int' if nextcloud_preview_preview_max_x is defined else 'string' }}"
+          value: "{{ nextcloud_preview_preview_max_x | default('null') }}"
```

Comment:
`is defined` is odd. All of our roles always work with defined variables (so that these variables have at least some default value). We don't deal with the possibility of "undefined" anywhere.

`| default('null')` also seems odd. Did you mean `null`, instead of `'null'`?

Reply:
Actually `null` is a value that NC gets. I change the type of value to `string`, this way the default value "null" and other integers can be used.

Reply:
I think I addressed this in 200e1e1002608883eeae42e4532538667625e21f